spot_ros2_control/README.md

```diff
@@ -24,6 +24,14 @@ You can then run the launchfile with the following command:
 ros2 launch spot_ros2_control spot_ros2_control.launch.py hardware_interface:=robot config_file:=path/to/spot_ros.yaml
 ```
 
+Joint level gains can also be specified in this same config file under the parameter names `k_q_p` and `k_qd_p`. If you do not specify these parameters, the default gains from the `spot-sdk` joint control examples are used during command streaming. The example below shows a valid configuration for a robot with an arm as each list contains 19 elements. More information on how these gains are used by Spot can be found [here](https://dev.bostondynamics.com/docs/concepts/joint_control/readme).
```

Comment:
Should there be a default config file included? Do we show an example with the CLI argument for the config?

Reply:
The CLI argument is the same as the command just above this paragraph (`config_file:=path/to/config.yaml'). I don't want there to be a default config file since a) a common use case for external users is to put their login info in the config file, which they are going to want to customize and b) there would have to be more logic in the launchfile to handle different default config files for robots with and without the arm since they have different #s of elements. If no config file is used then the defaults are picked up by the hardware interface